Tasks/Checkout/task.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
remove this file, this is an example for you, not for final check-in  :D

Reply:
removed